### DISCUSSION TITLE
What's the easiest way to create a contact form?

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [nunocoracao/blowfish#538] [Q&A]
I'm wondering what the easiest way to put a contact form is.

**Comment by nunocoracao** [comment]
Hello @SamGreenwood1 can you define a contact form here? 
Do you want someone to be able to press a button to send you an email? 
Create an UI experience for that?
Integrate with a service that does it for you?

Generically speaking you can use the override mechanism in Hugo to override (or create) new partials/shorcodes on-top of any theme. If you want something custom to your needs this might be a way to move forward. There might be simpler solutions using the theme features but I need to understand the use-case better.

**Reply by SamGreenwood1** [reply]
I was thinking about using Formspree, do you have any suggestions? How would I do it? I have experience using it but I've only used it with Hugo themes it was already built into.

**Reply by nunocoracao** [reply]
I would say to check those themes and see how they integrate it. It would be possible to implement that as an override to the theme on your site.
